### ISSUE TITLE
dicomviewer-master with nextcloud 15

### ISSUE THREAD

**Issue by XSBen** [ayselafsar/dicomviewer#31]
### Steps to reproduce
1. Install nextcloud 15
2. Install dicomviewer @ 75119eaf5dee42534701a44ed38ce054f11528c7

### Expected behaviour
See "Open in dicom viewer" when I click on directory that contains "*.dcm" files.

### Actual behaviour
See nothing

### Server configuration

**Operating system**:
Linux Debian Stretch

**Web server:**
nginx 1.14

**Database:**
postgresql 9.6

**PHP version:**
php 7.2.13

**Nextcloud version:** (see Nextcloud admin page)
15.0

**Where did you install Nextcloud from:**
From source

**List of activated apps:**
Enabled:
  - accessibility: 1.1.0
  - activity: 2.8.2
  - admin_audit: 1.5.0
  - bruteforcesettings: 1.3.0
  - cloud_federation_api: 0.1.0
  - comments: 1.5.0
  - dav: 1.8.0
  - dicomviewer: 1.0.2
  - end_to_end_encryption: 1.1.0
  - federatedfilesharing: 1.5.0
  - federation: 1.5.0
  - files: 1.10.0
  - files_antivirus: 2.0.0
  - files_downloadactivity: 1.4.0
  - files_pdfviewer: 1.4.0
  - files_sharing: 1.7.0
  - files_texteditor: 2.7.0
  - files_trashbin: 1.5.0
  - files_versions: 1.8.0
  - files_videoplayer: 1.4.0
  - firstrunwizard: 2.4.0
  - gallery: 18.2.0
  - impersonate: 1.2.0
  - logreader: 2.0.0
  - lookup_server_connector: 1.3.0
  - nextcloud_announcements: 1.4.0
  - notifications: 2.3.0
  - oauth2: 1.3.0
  - password_policy: 1.5.0
  - piwik: 0.5.0
  - provisioning_api: 1.5.0
  - serverinfo: 1.5.0
  - sharebymail: 1.5.0
  - systemtags: 1.5.0
  - theming: 1.6.0
  - twofactor_backupcodes: 1.4.1
  - workflowengine: 1.5.0

**Nextcloud configuration:**

```
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "partage.baclesse.fr"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "https:\/\/partage.baclesse.fr",
        "dbtype": "pgsql",
        "version": "15.0.0.10",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "UTC",
        "installed": true,
        "mail_smtpmode": "smtp",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "25",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "theme": "",
        "loglevel": 2,
        "maintenance": false,
        "updater.release.channel": "production",
        "trashbin_retention_obligation": 8,
        "twofactor_enforced": "true",
        "twofactor_enforced_groups": [
            "2FA"
        ],
        "twofactor_enforced_excluded_groups": [],
        "mail_sendmailmode": "smtp"
    }
}

```

### Client configuration
**Browser:**
Firefox

**Operating system:**

#### Nextcloud log (data/owncloud.log)
```
Insert your Nextcloud log here
```

#### Browser log
```
Content Security Policy: La directive « child-src » est obsolète. Veuillez utiliser la directive « worker-src » pour contrôler des workers ou la directive « frame-src » pour contrôler des images.
Content Security Policy: Les paramètres de la page ont empêché le chargement d’une ressource à eval (« script-src »).
Content Security Policy: Les paramètres de la page ont empêché le chargement d’une ressource à inline (« script-src »).
files:1:1
JQMIGRATE: Migrate is installed, version 1.4.0 core.js:7:542
Content Security Policy: Les paramètres de la page ont empêché le chargement d’une ressource à eval (« script-src »).
Échec du chargement pour l’élément <script> dont la source est « https://partage.baclesse.fr/apps/dicomviewer/js/app.bundle.js?v=1c08e88a-2 ». files:56:1
window.controllers/Controllers est obsolète. Veuillez ne pas l’utiliser pour détecter l’agent utilisateur (UA). merged.js:2176
Logging system initialised at Thu Dec 27 2018 10:19:00 GMT+0100 (heure normale d’Europe centrale) common.js:206:17
New search handler registered search.js:67:4
querying Whats New data was successful: nocontent merged-template-prepend.js:4616:4
undefined
```

**Comment by ayselafsar** [comment]
That is the same exception thrown by the old version. You probably didn't clear your browser history/cache, so your browser uses the old version from its cache.

Please make sure you did those:
1. Build dicom viewer with the increased version in appinfo/info.xml
2. Clear browser history/cache before testing

I plan releasing the new version soon.

**Comment by ayselafsar** [comment]
This is a duplicate of #29 and fixed in v1.1.0

**Comment by XSBen** [comment]
It's perfect !
Thank you very much :)
